pkg/services/pluginsettings/models.go

```diff
@@ -49,3 +54,63 @@ type GetByPluginIDArgs struct {
 	PluginID string
 	OrgID    int64
 }
+
+type PluginSetting struct {
+	Id             int64
```

Comment:
it would be amazing, if these would be change to capital, but not  necessarily on this PR